proof/refine/ARM/Reply_R.thy

```diff
@@ -13,33 +13,25 @@ defs replyUnlink_assertion_def:
     \<equiv> \<lambda>replyPtr state s. state = BlockedOnReply (Some replyPtr)
                           \<or> (\<exists>ep d. state = BlockedOnReceive ep d (Some replyPtr))"
 
-crunches setReplyTCB
-  for pred_tcb_at'[wp]: "\<lambda>s. P (pred_tcb_at' proj test t s)"
-  and tcb_at'[wp]: "\<lambda>s. P (tcb_at' t s)"
-  and ksReadyQueues[wp]: "\<lambda>s. P (ksReadyQueues s)"
-  and ksSchedulerAction[wp]: "\<lambda>s. P (ksSchedulerAction s)"
-  and valid_queues[wp]: "valid_queues"
-  and reply_at'[wp]: "\<lambda>s. P (reply_at' rp s)"
-
```

Comment:
This used to generate a bunch more of `updateReply` lemmas that are not included in the merged version below. My proof uses one of those and now I want to revive these crunches for `updateReply`. I am just wondering if other people might experience the same while rebasing.

Reply:
Sure thing. Go for it! Just make a Pull Request?

Reply:
Yes, the PR is up now :)